adapter/internal/oasparser/model/mgw_swagger.go

```diff
@@ -221,6 +252,15 @@ func (swagger *MgwSwagger) setXThrottlingTier() {
 	}
 }
 
+// resolve authHeader from swagger vendor extension map
+func resolveAuthHeader(vendorExtensions map[string]interface{}) string {
+	if xAuthHeader, ok := vendorExtensions[xAuthHeader]; ok {
+		// If x-wso2-auth-header is presen,
```

Comment:
typo in the comment   (presen -> present)

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/1785/commits/5e7db9dba2577979211d7713f6690403ca650326